### PR TITLE
Deploy multi-cloud services in soda-multi-cloud namespace

### LINKED ISSUE #1202: Fix review comments for multicloud-ha
**Issue/Feature Description:**
- IMHO Sensitive info like username and password should be stored in Secrets.
Ref: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

- The mount can be ignore by using kubernetes logging pipeline

- we can apply multiple resources in a directory and sub directory by
kubectl apply -f <directory> --recursive

- I think its not good choice to provide additional privileges to default service account.
A good choice would be to create a namespace , add a service account and give access to that service account.
Also same Service account need to be get used for add deployment of application trying to access kubernetes APIs by github.com/micro/go-plugins/registry/kubernetes/v2

**Why this issue to fixed / feature is needed(give scenarios or use cases):**

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**
Deploy multi-cloud services in soda-multi-cloud namespace
Add delete name space script. 

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
Improved management of soda-multi-cloud pods.
easy to delete , and localized env for multi-cloud pods using namespaces

**Which issue(s) this PR fixes**:

Fixes #1202 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
@AmitRoushan , @kumarashit  Pls review.

**Comment by rajat-soda** [comment]
@AmitRoushan : for now executing the deployments and service yamls individually.. for some time till HA stabilizes .
Will move to --recursive in subsequent PR.

File: K8s/k8s-mongo.sh [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-
-kubectl apply -f mongo-pvc-0.yaml
-kubectl apply -f mongo-pvc-1.yaml
-kubectl apply -f mongo-pvc-2.yaml
-
-
-kubectl apply -f mongo-service-0.yaml
-kubectl apply -f mongo-service-1.yaml
-kubectl apply -f mongo-service-2.yaml
+kubectl apply -f namespace.yaml
```
Comment (AmitRoushan) [inline K8s/k8s-mongo.sh:1]: we can use ```kubectl create namespace soda-multi-cloud```

**Review by AmitRoushan — CHANGES_REQUESTED** [review]

**Comment by AmitRoushan** [comment]
Improvement pointer: We can optimize the yamls by adding ```namespace``` attribute.

File: K8s/k8s-mongo.sh (hunk 1 above)
Comment (rajat-soda) [reply K8s/k8s-mongo.sh]: Done

File: K8s/k8s-cleanup.sh [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
-kubectl delete configmap tidb-sql 
-kubectl delete configmap s3-sql 
-kubectl delete configmap yig-sql 
+kubectl delete pvc mongo-0-pv-claim -n soda-multi-cloud
+kubectl delete pvc mongo-1-pv-claim -n soda-multi-cloud
+kubectl delete pvc mongo-2-pv-claim -n soda-multi-cloud
 
+kubectl delete pv mongo-0-pv-volume -n soda-multi-cloud
+kubectl delete pv mongo-1-pv-volume -n soda-multi-cloud
+kubectl delete pv mongo-2-pv-volume -n soda-multi-cloud
```
[the remark this answers is not in the document]
Comment (rajat-soda) [reply K8s/k8s-cleanup.sh:48]: Done

**Review by AmitRoushan — APPROVED** [review]
LGTM

File: K8s/api-deployment.yaml [hunk 3]
```diff
@@ -53,7 +53,7 @@ spec:
             - name: OS_AUTH_AUTHSTRATEGY
               value: keystone
             - name: OS_AUTH_URL
-              value: http://192.168.20.108/identity
+              value: http://KEYSTONE_IP/identity
```
Comment (kumarashit) [inline K8s/api-deployment.yaml:56]: does YAML supports plain variable replacement?

Comment (asifdxtreme) [reply K8s/api-deployment.yaml:56]: it needs something like `${ENV_VAR}`

Comment (rajat-soda) [reply K8s/api-deployment.yaml:56]: Yes.

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by asifdxtreme — APPROVED** [review]
LGTM

**Review by NajmudheenCT — APPROVED** [review]
LGTM

**Pull request merged**